os/board/common/crashdump.c

```diff
@@ -201,7 +280,7 @@ static int send_ramdump(void)
 
 static int send_fsdump(void)
 {
-	int i;
+	int i = 0;
```

Comment:
Is this initialization needed? `i` will be always set from `get_userfs_address_size()`.

Reply:
I follow this as a general code writing practice to avoid garbage values.
If this doesn't have any disadvantages, I'd like to keep this please.
Line 322 has been written in the same way.